defaults/main.yml

```diff
@@ -85,9 +85,87 @@ bbb_dialin_mask_caller: false
 bbb_dialin_overwrite_footer: false
 bbb_dialin_footer: <br><br>To join this meeting by phone, dial:<br>  %%DIALNUM%%<br>Then enter %%CONFNUM%% as the conference PIN number.
 
+bbb_meeting_inactivity_timeout_minutes: 10
 bbb_default_presentation: ${bigbluebutton.web.serverURL}/default.pdf
 
+bbb_custom_presentation: "default.pdf"
```

Comment:
Hmm, I am not sure about this. I would rather prefer to (by default) not touch this file and only do this, when some variables is set. What is you opinion about that?

Reply:
Okay, I maybe see the point of doing this: It allows to change back to the original version of the presentation...

Reply:
To stick closer to the BBB default, yes this makes sense.

Reply:
`taks/config.yml`
```yaml
- name: set default presentation
  become: true
  copy:
    src: "{{ bbb_custom_presentation }}"
    dest: "/var/www/bigbluebutton-default/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
    mode: 0644
  when: bbb_custom_presentation is defined and bbb_custom_presentation_name is defined
```
and
`defaults/main.yml`
```yaml
# bbb_custom_presentation: "default.pdf"
# set this to the name the default presentation on the host should have.
# if its not default.pdf moderators are able to delete the default presentation. Thats sometimes helpful and works fine.
# bbb_custom_presentation_name: "default.pdf"
```

Reply:
also updating the README